src/util-ioctl.c

```diff
@@ -55,26 +55,16 @@
  *
  * \param Name of a network interface
  */
-static int GetIfaceMaxHWHeaderLength(const char *pcap_dev)
+static int GetIfaceMaxHWHeaderLength(const char *dev)
```

Comment:
Why this renaming ?

Reply:
Renaming was done following this review comment: https://github.com/OISF/suricata/pull/9372#discussion_r1290352129